proto/tendermint/abci/types.proto

```diff
@@ -258,6 +297,32 @@ message ResponseApplySnapshotChunk {
   }
 }
 
+message ResponseExtendVote {
+  tendermint.types.VoteExtension vote_extension = 1;
+}
+
+message ResponseVerifyVoteExtension {
+  Result result = 1;
+
+  enum Result {
+    UNKNOWN = 0;  // Unknown result, treat as ACCEPT by default
+    ACCEPT  = 1;  // Vote extension verified, include the vote
+    SLASH   = 2;  // Vote extension verification aborted, continue but slash validator
```

Comment:
What does this mean for tendermint? i.e. what does tendermint do when it gets a `SLASH` response?